scripts/openstack_compute_install.sh

```diff
@@ -8,6 +8,9 @@ salt "cmp*" state.apply
 # Provision opencontrail virtual routers
 salt -C 'I@opencontrail:control:id:1' cmd.run "/usr/share/contrail-utils/provision_vrouter.py --host_name cmp01 --host_ip 172.16.10.105 --api_server_ip 172.16.10.254 --oper add --admin_user admin --admin_password workshop --admin_tenant_name admin"
 
+# Enables eth1 interface on compute nodes
+salt 'cmp*' cmd.run "ip link set up eth1"﻿⁠⁠⁠⁠
```

Comment:
As discussed with @simonpasquier, will try to find a way to make this even more robust by querying contrail-status before going on with the following reboot command so that we are sure the configuration is resilient